packages/admin/cms-admin/src/generator/future/generateGrid.ts

```diff
@@ -356,17 +363,36 @@ export function generateGrid(
                         sortable: !sortFields.includes(column.name) ? `false` : undefined,
                         valueGetter: column.valueGetter,
                         valueOptions: column.valueOptions,
-                        width: column.width ? String(column.width) : "150",
                         renderCell: column.renderCell,
-                    }),
-                )
+                        width: typeof column.width === "undefined" ? undefined : String(column.width),
+                        flex: typeof column.flex === "undefined" ? undefined : String(column.flex),
+                    };
+
+                    if (typeof column.width === "undefined") {
```

Comment:
I was thinking of:

```ts
if (typeof column.width === "undefined" && typeof column.minWidth === "undefined" && typeof column.maxWidth === "undefined" && typeof column.flex === "undefined") {
  // No custom width values, use default values
  columnDefinition.flex = "1";
  columnDefinition.minWidth = "150";
} else {
  // Apply values as defined in column defition
}
```

What do you think?

Reply:
I don't think we should remove all default values, when just one other value if defined...

I might just want to set a `maxWidth` to a `column` but that would then cause the column to lose it's `minWidth` and the ability to stretch into the available, free space. Then I'd have to manually add `minWidth: 150` and `flex: 1` to every column where I want a `maxWidth`.